objects/obj_controller/Draw_64.gml

```diff
@@ -2,35 +2,42 @@
 
 draw_set_font(font_press_start_sm)
 
-if global.game_state == GameState.Playing
+if current_game_state == GameState.Playing
 {
-	// score
-	var score_x = 120
-	var score_y = 66
-	draw_sprite(spr_scoreboard, 0, score_x, score_y)
+	draw_set_color(c_white)
+	draw_set_halign(fa_left);
+
+	var distance_x = 40
+	var distance_y = 80
+	
+	draw_sprite(spr_panel_score, 0, distance_x, distance_y)
 	draw_set_font(font_press_start_md)
-	draw_text(score_x, score_y, number_format_commas(global.score))
+	draw_text(distance_x+70, distance_y+22, number_format_commas(round(current_distance)) + "M")
 }
 
 // show game over screen
-if global.game_state == GameState.Finished
+if current_game_state == GameState.Finished
 {
 	// blue background
-	draw_sprite_stretched_ext(spr_overlay_blue, 0, 0, 0, room_width, room_height, image_blend, 0.75)
+	draw_sprite_stretched_ext(spr_color_light_blue, 0, 0, 0, room_width, room_height, image_blend, 0.80)
```

Comment:
is this sprite light blue or just white?

Reply:
Its a light blue from Pulse UI `#F2F7FF`